runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -155,13 +161,19 @@ public VersionLockedObject(Supplier<T> newObjectFn,
      * gc and external object access is needed.
      */
     public void gc(long trimMark) {
+        Timer.Context vloGcDuration = VloMetricsHelper.getVloGcContext();
```

Comment:
Can you move this and the lock acquisition to within the try lock? To be consistent with other places in the same file where the lock is acquired in the lock.

Reply:
done